sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -237,11 +237,10 @@ private ParameterBindingData ConvertBlobInputToParameterBindingData(BlobBaseClie
         private ParameterBindingData CreateParameterBindingData(string connection, string blobName, string containerName)
         {
             string connectionName = !string.IsNullOrEmpty(connection) ? _nameResolver.ResolveWholeString(connection) : string.Empty;
-            var connectionSection = _blobServiceClientProvider.GetWebJobsConnectionStringSection(connectionName);
 
             var blobDetails = new BlobParameterBindingDataContent()
             {
-                Connection = connectionSection.Key,
+                Connection = connectionName,
```

Comment:
What are the impacts of this change? Will it work fine with older worker extensions?

Reply:
Yup it should still work with older versions, the dotnet worker is doing the same work [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.Storage.Blobs/src/Config/BlobStorageBindingOptionsSetup.cs); and there hasn't been an issue with plain connections like "MyConnection" - the more complex names like "MyApp:MyConnection" are broken today. Both scenarios should work with this change so I don't expect breaking changes. If there are any suggestions for more validation, I'm happy to do more testing/checks (so far I've tested various connection names with the local package and its working as expected)

Reply:
This is however a breaking change for anyone that is ever binding to ParameterBindingData outside of the dotnet worker, so even though that should "never" happen, we should treat it as such.